tests/integration/helm_default_installation_test.go

```diff
@@ -300,22 +300,19 @@ func Test_Helm_Default_FluentD_Metadata(t *testing.T) {
 		Assess("expected container log metadata is present", stepfuncs.WaitUntilExpectedLogsPresent(
 			logsGeneratorCount,
 			map[string]string{
-				"cluster":         "kubernetes",
-				"namespace":       internal.LogsGeneratorName,
-				"pod_labels_app":  internal.LogsGeneratorName,
-				"container":       internal.LogsGeneratorName,
-				"deployment":      internal.LogsGeneratorName,
-				"replicaset":      "",
-				"namespace_id":    "",
-				"pod":             "",
-				"pod_id":          "",
-				"container_id":    "",
-				"host":            "",
-				"master_url":      "",
-				"node":            "",
-				"_sourceName":     "",
-				"_sourceCategory": "",
-				"_sourceHost":     "",
```

Comment:
Why do we removed some of the expected metadata?

Reply:
the expected metadata changes for otelcol, it has an additional entry and a few other changes:
`"_collector":       "kubernetes"`

Reply:
I'm curious what happened with `_source*`, these are pretty crucial fields and we should have it in sumo :grimacing:

Reply:
got it, I have added those `_source*`, fields to the tests and pushed the changes.